socrata/socrata_upload.py

```diff
@@ -10,6 +10,14 @@
 from pyathena import connect
 from pyathena.pandas.cursor import PandasCursor
 
+# Create a session object so HTTP requests can be pooled
+s = requests.Session()
```

Comment:
**[Nitpick, non-blocking]** I dislike single-character variable names in Python because they often conflict with debugger commands (in this case [`s` for `s(tep)`](https://docs.python.org/3/library/pdb.html#pdbcommand-step)). Plus, a word is typically more descriptive than a single letter anyway:

```suggestion
session = requests.Session()
```